templates/satspay/display.html

```diff
@@ -1,329 +1,210 @@
-                </q-tooltip>
-              </q-btn>
-              <q-btn
-                flat
-                v-else
-                @click="payOnchain"
-                style="color: primary; width: 100%"
-                label="onchain⛓️"
-              >
-                <q-tooltip> pay onchain </q-tooltip>
-              </q-btn>
-            </div>
-          </div>
-          <q-separator></q-separator>
-        </div>
-      </div>
-    </q-card>
-    <q-card class="q-pa-lg" v-if="lnbtc">
-      <q-card-section class="q-pa-none">
-        <div class="row items-center q-mt-sm">
-          <div class="col-md-2 col-sm-0"></div>
-          <div class="col-md-8 col-sm-12">
-            <div v-if="!charge.timeLeft && !charge.paid">
-              <q-icon
-                name="block"
-                style="color: #ccc; font-size: 21.4em"
-              ></q-icon>
-            </div>
-            <div v-else-if="charge.paid">
-              <q-icon
-                name="check"
-                style="color: green; font-size: 21.4em"
-                class="fit"
-              ></q-icon>
-              <div class="row text-center q-mt-lg">
-                <div class="col text-center">
-                  <q-btn
-                    outline
-                    v-if="charge.webhook"
-                    :loading="charge.paid"
-                    :percentage="timer"
-                    type="a"
-                    :href="charge.completelink"
-                    :label="charge.completelinktext"
-                    >{%raw%}<template v-slot:loading>
-                      {{charge.completelinktext}} </template
-                    >{%endraw%}</q-btn
-                  >
-                  <p class="q-pt-md">Redirecting after 5 seconds</p>
+            </q-item-section>
+          </q-item>
+        </q-list>
+      </q-card-section>
+      <q-card-section>
+        <q-tabs
+          v-model="tab"
+          dense
+          class="text-grey"
+          active-color="primary"
+          indicator-color="primary"
+          align="justify"
+          narrow-indicator
+          inline-label
+        >
+          <q-tab
+            v-if="charge.onchainaddress"
+            name="uqr"
+            icon="qr_code"
+            label="UQR (BIP21)"
+          ></q-tab>
+          <q-tab
+            v-if="charge.payment_request"
+            name="ln"
+            icon="bolt"
+            label="Lightning"
+          ></q-tab>
+          <q-tab
+            v-if="charge.onchainaddress"
+            name="btc"
+            icon="link"
+            label="Onchain"
+          ></q-tab>
+        </q-tabs>
+
+        <q-separator></q-separator>
+        <q-tab-panels v-model="tab" animated style="background: none">
+          <q-tab-panel name="uqr">
+            <div class="row justify-center q-mt-sm">
+              <div class="col-sm-10 col-md-8">
+                <div v-if="!charge.timeLeft && !charge.paid">
```

Comment:
- this conditions repeats 3 times (once in each tab)
- it should be moved on the `<q-tab` level

```html
<div v-if="!charge.timeLeft && !charge.paid">
    <q-icon
      name="block"
      style="color: #ccc; font-size: 21.4em"
    ></q-icon>
</div>
<div v-else-if="charge.paid">
    <satspay-paid :charge="charge" :timer="timer"></satspay-paid>
</div>
<q-tabs v-else ...>
   ...
</q-tabs>
```